docs/versions/v26.4.5/guide/plugins/caching/configuration.md

```
@@ -0,0 +1,126 @@
+import org.pipelineframework.annotation.PipelinePlugin;
+
+@PipelinePlugin("cache")
+public class CachePluginHost {
+}
+```
+
+## Application properties
+
+Cache plugin service properties:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)